### PR TITLE
File Added With Changes: Assignment 6 

### PR DESCRIPTION
**Pull request by akash19972306**

### CODE REVIEW

**Review by shivnanda6272 — APPROVED** [review]

File: akash1/HTML/ASSIn5/test1.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+         <meta http-equiv="X-UA-Compatible" content="IE=edge">
+         <meta name="viewport" content="width=device-width, initial-scale=1.0">
+         <title>webpage</title>
+         <link rel="stylesheet" href="style1.css">
+         <link href="https://fonts.googleapis.com/css2?family=Baloo+Bhaina+2&display=swap" rel="stylesheet">
+   </head>
+    <body>
+        <div class="first">
+            <div class="second"><strong>Akash Kadam</strong></div>
+        </div>
```
Comment (Pranjali1999) [inline akash1/HTML/ASSIn5/test1.html:14]: remove a closing div tag.

**Review by Pranjali1999 — CHANGES_REQUESTED** [review]

File: akash1/HTML/ASSIn5/test1.html (hunk 1 above)
Comment (akash19972306) [reply akash1/HTML/ASSIn5/test1.html:14]: i have to use two  <div> tag
that is tag inside tag so why i have to remove that tag

**Review by Pranjali1999 — APPROVED** [review]

**Review by akash19972306 — COMMENTED** [review]
ok

**Review by vinod2406 — APPROVED** [review]
ok

File: akash1/HTML/ASSIn1/assignment1.Q2.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Q2</title>
+    </head>
+    <body>       
+        <h1>HTML</h1>
+        <h3>INTRODUCTION</h3>
+        <p>HTML is a standred markup language for Web Pages.</p>
+            <h1>HTML Editor<h1>
```
Comment (SnehaMakade) [inline akash1/HTML/ASSIn1/assignment1.Q2.html:22]: indentation is not proper

File: akash1/HTML/ASSIn4/tableQ2.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>tableQ2</title>
+        <style>
+            table {
+            border: 10px solid gray;
+            border-collapse: collapse;
+              }
+            th,td {
```
Comment (SnehaMakade) [inline akash1/HTML/ASSIn4/tableQ2.html]: always add space after comma

File: akash1/HTML/ASSIn6/form6.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        <form>
+            <div class="block1">
+                <h1>MI-9 Internal HTML Assignment.</h1>
+            </div>
+            <hr />
+            <div class="block2">
+                <h2><em>Personal details</em></h2>
+            </div>
+            <label>First Name:</label>
+            <input type="text" name="fname" placeholder="firstname" /><br /><br />
```
Comment (SnehaMakade) [inline akash1/HTML/ASSIn6/form6.html]: placeholders 1st letter should be capital so make this change everywhere

File: akash1/HTML/ASSIn6/form6.html [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+            <input type="radio" id="26-40" name="age" value="26-40" />
+            <label for="26-40">26-40</label>
+            <div class="block3">
+                <h2>Academic details</h2>
+            </div>
+            <label for="age"> Skill Set:</label>
+            <input type="checkbox" id="HTML" name="skillset" value="HTML" />
+            <label for="vehicle1">HTML</label>
+            <input type="checkbox" id="Javascript" name="skillset" value="Javascript" />
+            <label for="vehicle2"> Javascript</label>
```
Comment (SnehaMakade) [inline akash1/HTML/ASSIn6/form6.html]: remove extra space

File: akash1/HTML/ASSIn6/form6.html [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+            <input type="checkbox" id="java" name="skillset" value="java" />
+            <label for="vehicle3"> Java</label>
```
Comment (SnehaMakade) [inline akash1/HTML/ASSIn6/form6.html]: remove extra space

File: akash1/HTML/ASSIn6/form6.html [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+            <input type="checkbox" id="oracle" name="skillset" value="oracle" />
+            <label for="vehicle3"> oracle</label>
```
Comment (SnehaMakade) [inline akash1/HTML/ASSIn6/form6.html]: remove extra space

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do the changes as per comments given

File: akash1/HTML/ASSIn1/assignment1.Q2.html (hunk 2 above)
Comment (akash19972306) [reply akash1/HTML/ASSIn1/assignment1.Q2.html:22]: this file is merged mam

File: akash1/HTML/ASSIn4/tableQ2.html (hunk 3 above)
Comment (akash19972306) [reply akash1/HTML/ASSIn4/tableQ2.html]: ok

File: akash1/HTML/ASSIn6/form6.html [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+        <form>
+            <div class="block1">
+                <h1>MI-9 Internal HTML Assignment.</h1>
+            </div>
+            <hr />
+            <div class="block2">
+                <h2><em>Personal details</em></h2>
+            </div>
+            <label>First Name:</label>
+            <input type="text" name="fname" placeholder="Firstname" /><br /><br />
```
Comment (SnehaMakade) [inline akash1/HTML/ASSIn6/form6.html]: placeholder should be like First name make this change in all input boxes

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
give proper placeholder name

**Review by SnehaMakade — APPROVED** [review]

**Pull request merged**